test-float-features.lisp

```diff
@@ -0,0 +1,107 @@
+(in-package #:cl-user)
+
+(defpackage #:float-features/tests
+  (:nicknames #:org.shirakumo.float-features/tests)
+  (:use #:cl))
+
+;;; to run (parachute:test :float-features/tests)
+(in-package #:float-features/tests)
+
+(parachute:define-test float-features-1
```

Comment:
Please come up with better names for these tests. `float-features-1` says absolutely nothing about what the test is supposed to actually test.

Reply:
ok

Reply:
done